test/subproc-specs.js

```diff
@@ -60,11 +60,23 @@ describe('SubProcess', function () {
     lines.should.include('bad_exit.sh');
     lines.should.contain('bigbuffer.js');
     lines.should.contain('echo.sh');
+    await subproc.stop();
   });
 
   describe('#start', function () {
+    /** @type {SubProcess} */
+    let s;
+
+    afterEach(async function() {
+      if (s) {
+        try {
+          await s.stop();
+        } catch {}
+      }
```

Comment:
shall we also assign `s` to null/undefined?

Reply:
👍